src/convertToObject.js

```diff
@@ -3,16 +3,26 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split('\n')
```

Comment:
better split by `;`,
now you can't parse inline styles

<img width="570" alt="image" src="https://user-images.githubusercontent.com/55802630/177805267-5f09a694-4fcf-4943-a771-8672ba7fe04f.png">

Reply:
yeah, but splitting by `\n` is totally unreliable, cause it is just formatting, and your function shouldn't depend on whether or not I formatted my styles correctly